docs/data-masking-function-list.md

```diff
@@ -0,0 +1,888 @@
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| str | No | The string to be masked | String |
+| mask_char | Yes | The masking character | String |
+
+The `str` contains a minimum of 14 or a maximum of 19 alphanumeric characters. 
+
+If the `mask_char` is not specified, the default is 'X'.
+
+### Returns
+
+A string with the selected characters masked by a specified `mask_char` or that parameter's default value. 
+
+An error occurs if the `str` parameter is not the correct length. A multi-byte character `str` is not masked.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_pan (gen_rnd_pan());
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------+
+    | mask_pan(gen_rnd_pan())            |
+    +------------------------------------+
+    | XXXXXXXXXXX2345                    |
+    +------------------------------------+
+    ```
+
+## mask_pan_relaxed(str [,mask_char])
+
+
+
+Returns a masked payment card Primary Account Number (PAN). The first six numbers and the last four numbers and the rest of the string masked by specified character or `X`.
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| str | No | The string to be masked | String |
+| mask_char | Yes | The masking character | String |
+
+The `str` must contain a minimum of 14 or a maximum of 19 alphanumeric characters. 
+
+If the `mask_char` is not specified, the default is 'X'.
+
+### Returns
+
+A string with the first six numbers and the last four numbers and the rest of the string masked by a specified `mask_char` or that parameter's default value (`X`). 
+
+An error occurs if the `str` parameter is not the correct length. A multi-byte character `str` is not masked.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_pan_relaxed(gen_rnd_pan());
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------------+
+    | mask_pan_relaxed(gen_rnd_pan())          |
+    +------------------------------------------+
+    | 520754XXXXXX4848                         |
+    +------------------------------------------+
+    ```
+
+## mask_ssn(str [,mask_char])
+
+
+
+Returns a masked United States Social Security Number(SSN). The mask replaces the SSN number with the specified character except for the last four digits.
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| str | No | The string to be masked | String |
+| mask_char | Yes | The masking character | String |
+
+The `str` accepts either of the following:
+
+* Nine integers, no separator symbol
+* Nine integers in the `***-**-****` pattern. The `-` is a separator character.
+
+If the `mask_char` is not specified, the default is `*`.
+
```

Comment:
mask_char can be a multibyte character in any character set (not necessarily identical to the character set of str)